src/main/java/com/adamk33n3r/runelite/watchdog/WatchdogPanel.java

```diff
@@ -269,12 +269,19 @@ private void createAlert(TriggerType triggerType) {
 
     private PluginPanel createPluginPanel(Alert alert) {
         if (alert instanceof ChatAlert) {
-            ChatAlert chatAlert = (ChatAlert) alert;
+            ChatAlert gameMessageAlert = (ChatAlert) alert;
             return AlertPanel.create(this.muxer, alert)
                 .addAlertDefaults(alert)
-                .addRegexMatcher(chatAlert, "Enter the message to trigger on...", "The message to trigger on. Supports glob (*)")
+                .addRegexMatcher(gameMessageAlert, "Enter the message to trigger on...", "The message to trigger on. Supports glob (*)")
                 .addLabel("<html><i>Note: Will not trigger on<br>player chat messages</i></html>")
                 .build();
+        } else if (alert instanceof PlayerChatAlert) {
+            PlayerChatAlert gameMessageAlert = (PlayerChatAlert) alert;
+            return AlertPanel.create(this.muxer, alert)
+                    .addAlertDefaults(alert)
+                    .addRegexMatcher(gameMessageAlert, "Enter the message to trigger on...", "The message to trigger on. Supports glob (*)")
+                    .addLabel("<html><i>Note: Will only trigger on<br>player chat messages</i></html>")
```

Comment:
I wonder if we can remove these labels from both of the types since it should now be clear with both available.

Reply:
I wasn't sure. Good call will do.